src/poliastro/twobody/events.py

```diff
@@ -25,7 +25,7 @@ def terminal(self):
     def last_t(self):
         return self._last_t * u.s
 
-    def __call__(self, t, u):
+    def __call__(self, t, u, k):
```

Comment:
Is this `k` ever used? :thinking:

Reply:
Not in the `LithobrakeEvent`, but potentially in other events! Also, makes the calling convention of the objective function and the events consistent.